docs/application/tizen-studio/common-tools/log-view.md

```diff
@@ -1,9 +1,5 @@
 # Checking Logs with Log View
 
-> **Note**
->
-> Log Viewer has been deprecated from Tizen Studio. Device Manager can be used to view the device logs.
```

Comment:
Question related with https://github.com/Samsung/tizen-docs/pull/1054/files/c4e12eeb558197a1d32940f8fe68bce0ec76f605..e97f2470598b0d5a0fc74bc2c178f3d078b6ba8f

If the "Log Viewer has been re-enabled in Tizen IDE." is removed from the release note,
I think this updated should be reverted back.
Would you please check?

Reply:
"Log Viewer has been re-enabled in Tizen IDE." is added back to the release notes